.gitlab-ci.yml

```diff
@@ -553,6 +559,49 @@ build/amd/clang/hip_wo_omp/release/shared:
     BUILD_HIP: "ON"
     BUILD_TYPE: "Release"
 
+# mpi job test debug shared
+build/nocuda/gcc/mpi/debug/shared:
+  <<: *default_build_with_test
+  extends:
+    - .quick_test_condition
+    - .use_gko-nocuda-gnu9-llvm8-intel
+  variables:
+    <<: *default_variables
+    BUILD_MPI: "ON"
+    MPI_AS_ROOT: "ON"
```

Comment:
Is this a requirement for Docker?

Reply:
Yes, to run within docker, you need to run mpi as root. As docker is self-contained, a catastrophic error does not hurt us.